external/CMakeLists.txt

```diff
@@ -273,7 +276,12 @@ if(CMAKE_CROSSCOMPILING)
     cryptoauthlib/lib/crypto/hashes/sha2_routines.c
     cryptoauthlib/lib/crypto/atca_crypto_sw_sha2.c
   )
-  target_link_libraries(cryptoauthlib asf4-drivers-min)
+  target_link_libraries(cryptoauthlib
+    PRIVATE
+      CMSIS
```

Comment:
why does cryptoauthlib even need any of those?

Reply:
I think it's because it's a wrapper around those libs.